cli/remotebazel/remotebazel.go

```diff
@@ -70,7 +73,7 @@ func consoleDeleteLines(n int) {
 type RunOpts struct {
 	Server            string
 	APIKey            string
-	Args              *commandline.BazelArgs
+	Args              []string
 	WorkspaceFilePath string
 	SidecarSocket     string
```

Comment:
looks like this just needed a target where it could download artifacts for, so i switched it over to using the server target and removed the sidecar socket altogether for now